setup.py

```diff
@@ -11,23 +11,24 @@
 # FOR A PARTICULAR PURPOSE.
 #
 ##############################################################################
-"""Setup
-"""
+version = '1.6.0'
```

Comment:
Gaaaa, well I normally don't update the version until release.  I dislike the extra dev dance.  But you're righ that I should have pushed master and the tag earlier....